app/objects/c_ability.py

```diff
@@ -66,7 +66,7 @@ def unique(self):
     def __init__(self, ability_id, tactic=None, technique_id=None, technique=None, name=None, test=None,
                  description=None, cleanup=None, executor=None, platform=None, payloads=None, parsers=None,
                  requirements=None, privilege=None, timeout=60, repeatable=False, buckets=None, access=None,
-                 variations=None, language=None, code=None, build_target=None, **kwargs):
+                 variations=None, language=None, code=None, build_target=None, additional_info=dict(), **kwargs):
```

Comment:
also, good practice to avoid mutable defaults in python

Reply:
Thanks, changed to None